roles/control_node/tasks/30_controller.yml

```diff
@@ -100,7 +100,7 @@
     name: "{{ item[1] }}"
   with_nested:
     - [ "awx", "{{ username }}" ]
```

Comment:
When we are populating controller with the same EEs that means we are importing it in awx user, I would imaging we only need the workshop user import here.